### PR TITLE
[backend] Include available category options in the helptext

### PR DESCRIPTION
**Pull request by harshalmittal4**
Minor update in PuppetForge backend to include available category options to the helptext and align it to the parent Backend.
@valeriocos @sduenas, please see if this is good and I will update the other two backends similarly.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -42,8 +42,7 @@
     import pypandoc
     long_description = pypandoc.convert(readme_md, 'rst')
 except (IOError, ImportError):
-    print("Warning: pypandoc module not found, or pandoc not installed. " +
-          "Using md instead of rst")
+    print("Warning: pypandoc module not found, or pandoc not installed. Using md instead of rst")
```
Comment (valeriocos) [inline setup.py]: I would leave this code as it is, which is the same of what is in [perceval](https://github.com/chaoss/grimoirelab-perceval/blob/master/setup.py#L46).

Comment (harshalmittal4) [reply setup.py]: Actually it is giving flake8 error, so thats why modified it..

Comment (harshalmittal4) [reply setup.py]: `./setup.py:45:75: W504 line break after binary operator`

Comment (harshalmittal4) [reply setup.py]: Ok, I'll remove the binary operator, let me know for any other changes.

Comment (valeriocos) [reply setup.py]: yes, perfect! please align the code to https://github.com/chaoss/grimoirelab-perceval/blob/master/setup.py#L46 , and put it in a different commit

Comment (valeriocos) [reply setup.py]: The other option is to leave as it is, but please add some explanation in the commit message. The point is that in the current status, there is no info about why this file has been modified. Thus, if in the future we check the git log, we will not know why this was done.

Comment (harshalmittal4) [reply setup.py]: Yes agreed, to keep a record, I have made it into a seperate commit @valeriocos

Comment (harshalmittal4) [reply setup.py]: Yes, agreed, to keep track, made a seperate commit @valeriocos.

**Comment by valeriocos** [comment]
Thank you @harshalmittal4 , I guess the perceval version of the [install_requires](https://github.com/chaoss/grimoirelab-perceval-puppet/blob/master/setup.py#L105) should be updated at some point, but there is no pip version that includes the recent change in [perceval](https://github.com/chaoss/grimoirelab-perceval/commit/7880dedc8ab83f77276fb44556dde9c76f0c2f2f).

Let's wait for @sduenas feedback

**Comment by harshalmittal4** [comment]
Yes, then maybe the PR can be merged after the latest pip release of percaval..

**Comment by sduenas** [comment]
@harshalmittal4 we decided we will update Perceval dependencies with the next release. People installing from sources should don't have problems of dependencies.

**Pull request merged**